### DISCUSSION TITLE
Livewire form input miss value when loading

### DISCUSSION THREAD

**Discussion by jocket1** [livewire/livewire#7611] [1. Help]
I have a form data and some input with type text
When I set the network of Chrome is slow 3g. The browser loading then I input some text after that it loads successfully then the text disappears. 

https://github.com/livewire/livewire/assets/154584353/2c36e08c-bffa-4c54-b802-e6b91ffe337a

**Comment by dinhhoa1994** [comment]
I also got the same error but don't know how to solve it.
After the livewire loaded, all the input I entered while the page was loading disappeared
